### ISSUE TITLE
Everyone is labeled as 10.1515

### ISSUE THREAD

**Issue by Daniel-Mietchen** [wpoa/Wikipedia-Cite-o-Meter#5]
Apart from prefixes listed on the starting page, calls like

```
http://tools.wmflabs.org/cite-o-meter/index.php?doip=10.1047
```

will result in a display "for Walter de Gruyter (10.1515)" due to https://github.com/dartar/Wikipedia-Cite-o-Meter/blob/master/index.php#L4 .

I think we had discussed a proper solution a while ago, but I don't remember what it was.

**Comment by dartar** [comment]
yes, that's annoying. I forgot what a more elegant solution looked like.
